content/blog/adopting-existing-cloud-resources-into-pulumi/index.md

```diff
@@ -24,15 +24,15 @@ We'll review referencing existing resources, and then dive deeper into how you c
 
 For referencing existing resources, Pulumi offers several tools.
 
-* The `.get` methods available on every resource let you [get all the details for a resource]({{< relref "/docs/reference/pkg/nodejs/pulumi/aws/ec2#Vpc-get" >}}) from the cloud provider based just on its `id`.
+* The `.get` methods available on every resource let you [get all the details for a resource]({{< relref "/docs/reference/pkg/aws/ec2/vpc/#look-up-an-existing-vpc-resource" >}}) from the cloud provider based just on its `id`.
```

Comment:
For `relref` links that have an anchor, there shouldn't be a `/` before the `#`.

```suggestion
* The `.get` methods available on every resource let you [get all the details for a resource]({{< relref "/docs/reference/pkg/aws/ec2/vpc#look-up-an-existing-vpc-resource" >}}) from the cloud provider based just on its `id`.
```